.circleci/config.yml

```diff
@@ -2,7 +2,7 @@ version: 2
 jobs:
   build:
     docker:
-      - image: circleci/openjdk:8-jdk
+      - image: circleci/openjdk:8-jdk-stretch
```

Comment:
looks like we used openjdk 10 with travis, any reason for the switch to 8 here?

Reply:
yeah, we had to switch to 10 because there is an open issue with 11
https://github.com/spotbugs/spotbugs/issues/756

Eventually I want to test versions 8+, but just going to switch it over for now.